sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobAsyncClient.java

```diff
@@ -88,7 +88,12 @@
      * value will be used.
      */
     public static final int BLOB_DEFAULT_HTBB_UPLOAD_BLOCK_SIZE = 8 * Constants.MB;
+    /**
+     * @deprecated Use {@link #BLOB_MAX_UPLOAD_BLOCK_SIZE_LONG}.
+     */
+    @Deprecated
     static final int BLOB_MAX_UPLOAD_BLOCK_SIZE = 100 * Constants.MB;
```

Comment:
This is scoped to `package private` shouldn't need deprecation.

Reply:
done